src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Collections/MutableIntervalTree`1.Node.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.CodeAnalysis.Shared.Collections;
 
 internal partial class MutableIntervalTree<T>
 {
-    protected sealed class Node
+    internal sealed class Node
```

Comment:
needed as it's part of the signature of the public GetEnumerator now (can't have generic struct enumerators without leaking details like this).